ros/bouncy/ubuntu/bionic/platform.yaml

```diff
@@ -10,4 +10,4 @@ platform:
     maintainer_name:
     arch: amd64
     type: distribution
-    version:
+    version: 2
```

Comment:
Could we update the ardent Dockerfiles with this? I know its EOL, but its helpful to keep the auto generate dockerfile in sync so that we only need the single `manifest` file to regenerate all folders in the library.

Reply:
Sure thing. Do you want me to update the ROS1 EOL Dockerfiles as well?

Reply:
If it's not to much trouble. Having updated `.config/ros1/platform.yaml.em` you should only need to delete the ros1 folders and run the `./create_dockerfolders.py` in the ros directory. You may need to toggle the comments of the EOL listings to ensure they get regenerated.

Reply:
Yeah that's no trouble at all. I'll update them all once we resolve the other pending question